testcontainers/src/core/container.rs

```diff
@@ -213,6 +225,13 @@ where
     }
 }
 
+/// Represents an output of `exec` command.
+/// `stdout` & `stderr` is represented as bytes because it might contain non-utf chars and responsibility should be on the caller end.
+pub struct ExecOutput {
+    pub stdout: Vec<u8>,
+    pub stderr: Vec<u8>,
+}
```

Comment:
Created separate type, because tuple with the same types could be confusing and erroneous. 

Also decided not to convert it to string, because it's up to caller if output can contain non-utf8